VERSION

```diff
@@ -1 +1 @@
-1.2
+1.3
```

Comment:
let's start using `MAJOR.MINOR.UPDATE` versioning.  So let's do `1.3.0` for this one.